get-victorops-schedule.ts

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import * as VictorOpsApiClient from 'victorops-api-client';
+import * as devOps from './helper-scripts/options/devOps.json';
+import * as _ from 'lodash';
+import { google, calendar_v3 } from 'googleapis';
+import { getAuthClient } from './lib/gauth';
+import { agents } from './logs/2022-08-01_devOps/support-shift-scheduler-input.json';
```

Comment:
This needs to be updated to point to an actually existing file, and it needs to have entries for every person that might be part of the victorops schedule (or otherwise it'll bork saying that an attendee doesn't have an email)